2023-03-30 00:30:34,575 [1] INFO  Photon.SocketServer.ApplicationBase - Application is stopping: AppId=SangoTest
2023-03-30 00:30:34,584 [12] ERROR Photon.SocketServer.ApplicationBase - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.SleepInternal(Int32 millisecondsTimeout)
   at System.Threading.Thread.Sleep(Int32 millisecondsTimeout)
   at FantasyOfSango.Threads.SyncPlayerTransformThreads.Update() in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Threads\SyncPlayerTransformThreads.cs:line 27
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
2023-03-30 00:30:34,597 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Shut Down
2023-03-30 00:30:34,597 [1] INFO  Photon.SocketServer.ApplicationBase - Application stop: AppId=SangoTest
2023-03-30 18:51:14,013 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Running
2023-03-30 18:51:14,036 [1] INFO  FantasyOfSango.SangoServer - Init all Component
2023-03-30 18:51:14,037 [1] INFO  Photon.SocketServer.ApplicationBase - Application start: AppId=SangoTest; AppPath=D:\FantasyOfSango\Server\TestServer\Photon_v4\FantasyOfSango_Photon\deploy\FantasyOfSango, Type=FantasyOfSango.SangoServer 
2023-03-30 18:51:26,426 [15] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-30 18:51:52,439 [13] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:50726, Data: F3-02-07-00-01-08-73-01-25-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-34-2E-36-36-38-33-35-30-32-2C-22-59-22-3A-31-30-2E-30-32-37-39-34-38-34-2C-22-5A-22-3A-31-33-32-2E-37-33-33-30-37-38-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-33-2E-37-38-34-31-31-38-37-2C-22-59-22-3A-39-2E-39-36-33-35-35-35-33-34-2C-22-5A-22-3A-31-33-31-2E-36-38-38-33-35-34-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-33-30-54-31-30-3A-35-31-3A-35-32-2E-32-33-34-34-39-37-31-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-30 18:52:10,597 [13] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:50726, Data: F3-02-07-00-01-08-73-01-25-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-38-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-33-39-36-31-37-39-32-2C-22-59-22-3A-39-2E-31-33-38-31-36-34-35-32-2C-22-5A-22-3A-31-33-36-2E-33-32-32-39-36-38-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-30-2E-32-35-39-31-35-35-33-2C-22-59-22-3A-38-2E-35-31-36-32-35-30-36-31-2C-22-5A-22-3A-31-33-35-2E-35-34-30-37-32-36-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-33-30-54-31-30-3A-35-32-3A-31-30-2E-34-35-33-33-33-38-37-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-30 18:52:13,229 [14] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:50726, Data: F3-02-07-00-01-08-73-01-24-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-33-39-36-31-37-39-32-2C-22-59-22-3A-39-2E-31-33-38-31-36-34-35-32-2C-22-5A-22-3A-31-33-36-2E-33-32-32-39-36-38-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-30-2E-32-35-39-31-35-35-33-2C-22-59-22-3A-38-2E-35-31-35-35-36-33-39-36-2C-22-5A-22-3A-31-33-35-2E-35-34-30-37-32-36-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-33-30-54-31-30-3A-35-32-3A-31-33-2E-30-38-36-39-37-38-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-30 18:52:26,659 [13] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-30 23:40:39,694 [1] INFO  Photon.SocketServer.ApplicationBase - Application is stopping: AppId=SangoTest
2023-03-30 23:40:39,708 [16] ERROR Photon.SocketServer.ApplicationBase - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.SleepInternal(Int32 millisecondsTimeout)
   at System.Threading.Thread.Sleep(Int32 millisecondsTimeout)
   at FantasyOfSango.Threads.SyncPlayerTransformThreads.Update() in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Threads\SyncPlayerTransformThreads.cs:line 27
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
2023-03-30 23:40:39,727 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Shut Down
2023-03-30 23:40:39,727 [1] INFO  Photon.SocketServer.ApplicationBase - Application stop: AppId=SangoTest
